.github/workflows/main.yaml

```diff
@@ -29,6 +29,27 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v4
 
+      - name: Fetch GH Sponsors
```

Comment:
Move this into it's own workflow, which can be set to run on a schedule rather than relying on people doing stuff